src/app/pages/StoryPage/index.jsx

```diff
@@ -70,11 +70,22 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
   const mostReadInitialData = path(['mostRead'], pageData);
-  const topStoriesInitialData = path(
+  const topStoriesInitialData = pathOr(
```

Comment:
Do we need to use `pathOr()`? Doesn't `path()` return `undefined` if it can't retrieve a value at that path?

Reply:
we do not necessarily need `pathOr` I just assumed in the case that it's missing an empty array would be returned for `topStories` or `features`

Reply:
@HarveyPeachey I have removed the use of `pathOr`